include/flexflow/parallel_tensor.h

```diff
@@ -99,13 +99,19 @@ struct ParallelTensorShape {
 
 std::ostream& operator<<(std::ostream&, ParallelTensorShape const &);
 
+typedef std::vector<ParallelTensorShape> ParallelTensorShapes;
```

Comment:
```suggestion
```

I'd suggest removing the type alias for simplicity (so people don't have to go figure out what it means--the type itself isn't that long)